content/backmatter.tex

```diff
@@ -522,6 +522,25 @@ \section{Version 1.5}
 \CONST{SHMEM\_MALLOC\_ATOMICS\_REMOTE} and \CONST{SHMEM\_MALLOC\_SIGNAL\_REMOTE}.
 \\ See Section \ref{subsec:shmmallochint} and \ref{subsec:library_constants}.
 %
+\item Changed the broadcast function to update the dest object on all PEs,
+including the root PE.
+\\ See Section \ref{subsec:shmem_broadcast}.
+%
+\item Clarified error codes returned from collectives.
+\\ See Section \ref{subsec:coll}.
+%
+\item Deprecated entire active-set-based collectives. Added the
+  team-based collective functions with return value to indicate successful
+  local completion: \FUNC{shmem\_sync},
```

Comment:
Can we split this up?
```
\item Deprecated active-set-based collectives.
\item Added team-based collective functions:
```
The return value doesn't seem to me like a detail that needs to be called out here?